### PR TITLE
add whitelist words.

### LINKED ISSUE #113: The whitelist issue
mashedpotatos

### PR DESCRIPTION
**Pull request by showierdata9978**
anyone can push

closes #113

### CODE REVIEW

**Review by JoshAtticus — COMMENTED** [review]
Uhhh, that needs to stay there

**Comment by showierdata9978** [comment]
what word???????

File: better_profanity/profanity_wordlist.txt [hunk 1]
```diff
@@ -8,7 +8,6 @@ arse
 arsehole
 aryan
 asanchez
-ass
```
Comment (JoshAtticus) [inline better_profanity/profanity_wordlist.txt:11]: Here

Comment (showierdata9978) [reply better_profanity/profanity_wordlist.txt:11]: i disagree

Comment (JoshAtticus) [reply better_profanity/profanity_wordlist.txt:11]: What does @MikeDev101 think of this?

**Comment by zedthehedgehog** [comment]
What words exactly?

**Comment by showierdata9978** [comment]
> What words exactly?

https://github.com/meower-media-co/Meower-Server/pull/114/files

**Comment by zedthehedgehog** [comment]
Maybe just ass by itself, other words, like asshole, asshat, etc. should continue to be blacklisted

**Comment by zedthehedgehog** [comment]
Maybe whitelist damn too, it's not that bad, and especially analytics, nobody, or other words that contain swears but aren't meant to mean a swear

**Comment by tnix100** [comment]
zed merged a branch in server 😱

**Comment by JoshAtticus** [comment]
> zed merged a branch in server 😱 

Tnix 😱

**Comment by mdwalters** [comment]
DISASTER TIME!!!!!!1!!!!!!!!!1111!!!!!1!!!

**Pull request merged**